deploy/ui/form.json

```
@@ -129,6 +129,38 @@
               "type": "password"
             }
           ]
+        },
+        {
+          "label": "Startup Strategy",
+          "type": "fieldGroup",
+          "jsonPath": "$.spec.startupStrategy",
+          "visible": true,
+          "fields": [
+            {
+              "type": "dropDown",
+              "required": true,
+              "jsonPath": "$.spec.startupStrategy.strategyName",
+              "originalJsonPath": "$.spec.startupStrategy.strategyName",
+              "description": "Specify the StartupStrategy to use. Do not enable the Controller strategy in an environment with a high-availability Business Central.",
+              "default": "ControllerBasedStartupStrategyOpenShiftStartupStrategy",
```

This looks not correct, there are used both strategies. Just use only one, that was default so far.
```suggestion
              "default": "OpenShiftStartupStrategy",
```

---

ok